Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStorePrivate.h

```diff
@@ -44,6 +44,7 @@ typedef NS_OPTIONS(NSUInteger, _WKWebsiteDataStoreFetchOptions) {
 + (NSSet<NSString *> *)_allWebsiteDataTypesIncludingPrivate;
 + (BOOL)_defaultDataStoreExists;
 + (void)_deleteDefaultDataStoreForTesting;
++ (WKWebsiteDataStore *)_dataStoreForNSAttributedString;
```

Comment:
This is only used internally, why is it in the SPI header?

Reply:
Which header should it be in? The WKWebsiteDataStoreInternal header?